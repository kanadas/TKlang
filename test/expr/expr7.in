(\a -> a) :: int * int -> int * int
