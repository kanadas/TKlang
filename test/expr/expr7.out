expr/expr7.in
7
